library/core/src/ptr/mod.rs

```diff
@@ -2552,8 +2552,9 @@ mod verify {
     }
 
     #[kani::proof_for_contract(align_offset)]
-    fn check_align_offset_17() {
-        let p = kani::any::<usize>() as *const [char; 17];
+    #[kani::solver(kissat)]
+    fn check_align_offset_5() {
+        let p = kani::any::<usize>() as *const [char; 5];
```

Comment:
Could you add a comment explain why 5 and our experience with the verification time blow up before we merge?